notebooks/3_subset_data.ipynb

```diff
@@ -0,0 +1,1358 @@
+{
```

Comment:
<p>The "assign new value to selection" is not done? (which might be good, there is already a lot here)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I'll add a short example to the tutorial to cover this as well.</p>